data-entry/invoice-editor/invoice-editor-flow/src/main/java/com/vaadin/invoice/editor/InvoiceEditor.java

```diff
@@ -200,20 +233,33 @@ public InvoiceEditor() {
                     return badge;
                 });
         grid.addEditColumn(Invoice::getOrderCompleted, statusRenderer)
-                .checkbox((item, newValue) -> displayNotification(
-                        "Order completed ", item, newValue.toString()))
-                .setHeader("Order completed");
+                .checkbox((item, newValue) -> {
+                    item.setOrderCompleted(newValue);
+                    displayNotification(
+                            "Order completed ", item, newValue.toString());
+                })
+                .setSortProperty("isOrderCompleted")
```

Comment:
If you set comparator, do you still need to set the sort property, or is this maybe unnecessary? (same for others)

Reply:
Removed.